MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -507,18 +497,5 @@ public protocol CarPlayNavigationDelegate {
      */
     @objc(carPlayNavigationViewControllerDidDismiss:byCanceling:)
     optional func carPlayNavigationViewControllerDidDismiss(_ carPlayNavigationViewController: CarPlayNavigationViewController, byCanceling canceled: Bool)
-    
-    /**
-     Called when the user arrives at the destination waypoint for a route leg.
-     
-     This method is called when the CarPlay navigation view controller arrives at the waypoint. You can implement this method to prevent the navigation view controller from automatically advancing to the next leg. For example, you can and show an interstitial sheet upon arrival and pause navigation by returning `false`, then continue the route when the user dismisses the sheet. If this method is unimplemented, the CarPlay navigation view controller automatically advances to the next leg when arriving at a waypoint.
-     
-     - postcondition: If you return `false` within this method, you must manually advance to the next leg: obtain the value of the `navigationService` and its `NavigationService.routeProgress` property, then increment the `RouteProgress.legIndex` property.
-     - parameter carPlayNavigationViewController: The CarPlay navigation view controller that has arrived at a waypoint.
-     - parameter waypoint: The waypoint that the user has arrived at.
-     - returns: True to automatically advance to the next leg, or false to remain on the now completed leg.
-     */
-    @objc(carPlayNavigationViewController:didArriveAtWaypoint:)
-    optional func carPlayNavigationViewController(_ carPlayNavigationViewController: CarPlayNavigationViewController, didArriveAt waypoint: Waypoint) -> Bool
```

Comment:
This is effectively the removal of `CarPlayManagerDelegate.carPlayNavigationViewControllerDidArrive(_:)`. Since the replacement of `NavigationServiceDelegate.navigationService(_:didArriveAt:)` isn’t obvious, keep a rump method around under the old name of `CarPlayManagerDelegate.carPlayNavigationViewControllerDidArrive(_:)` and add a message directing the developer to use `NavigationServiceDelegate.navigationService(_:didArriveAt:)` instead.